tony-core/src/main/java/com/linkedin/tony/ApplicationMaster.java

```diff
@@ -353,7 +354,13 @@ private boolean run(String[] args) throws IOException {
       // Prepare for retryCount.
       reset();
       LOG.info("Retrying, remaining retry count" + amRetryCount);
+
       amRetryCount -= 1;
+      numAMRetries += 1;
```

Comment:
prefer to start from 0, move the +1 after next line

Reply:
Ok, so set it to 0 within the scope of the method? Not line 118?

Reply:
118 should be ok, just move this to the nextline